qa/pull-tester/rpc-tests.py

```diff
@@ -83,6 +83,7 @@
 
 #Tests
 testScripts = [
+    'bip68-112-113-p2p.py',
```

Comment:
no, this is a softfork and certainly until there is activation, I want to test for regressions as a standard part of the CI process.